src/test/acceptance/features/choose-channel-for-code.feature

```diff
@@ -0,0 +1,25 @@
+Feature: We're going to send you a code
+  In order to apply for the HTBHF programme
+  As a potential claimant
+  I want to select text or email to receive my code
+
+  Background:
+    Given I have entered my details up to the choose channel for code page
+
+  Scenario: Text and Email are displayed
+    Then Text and Email options are displayed on the choose channel for code page
+
+  Scenario: Do not select an option
+    When I do not select an option
+    And I click continue
+    Then I am informed that I need to select an option for choose channel for code
+
+  Scenario: Select the Text option
+    When I select Text to how would I like to receive my code
```

Comment:
Minor suggested wording change: `When I answer 'Text' to how would I like to receive my code`

Reply:
Or maybe `When I choose Text as the method to receive the code`. I do like the word method rather than channel for this page, I think we should run this past Sam before we go too far with this story and have loads of renaming to do.

Reply:
Went for `I select Text as the method to receive the code` as the use of `I select` is used for both are you pregnant and do you live in Scotland.